packages/@aws-cdk/aws-ecs/README.md

```diff
@@ -148,7 +148,7 @@ const ec2TaskDefinition = new ecs.Ec2TaskDefinition(this, 'TaskDef', {
 
 const container = ec2TaskDefinition.addContainer("WebContainer", {
   // Use an image from DockerHub
-  image: ecs.ContainerImage.fromDockerHub("amazon/amazon-ecs-sample"),
+  image: ecs.ContainerImage.fromInternet("amazon/amazon-ecs-sample"),
```

Comment:
How about we call this `fromRegistry` ("registry" is the official term in the dockerverse)

Reply:
ECR is also a registry but we don't want to include that. But maybe that ambiguity is okay.

Reply:
I think it is okay, especially given we will have both “fromEcr” and “fromRegistry” next to each other which disambiguates